src/components/reusable/sideDrawer/sideDrawer.scss

```diff
@@ -174,15 +164,18 @@ h1 {
       transform: none;
 
       &::backdrop {
-        background-color: rgba(0, 0, 0, 0.4);
+        background: var(--kd-color--neutral-dark-opacity-50, rgba(61, 60, 60, 0.5));
+        backdrop-filter: blur(0.10000000149011612px);
```

Comment:
Curious, where did this blur come from?

Reply:
rounded up to 1px, blur is more prominent now
<img width="1254" alt="Screenshot 2024-12-04 at 9 26 55 AM" src="https://github.com/user-attachments/assets/a98fa758-79ad-41ee-8703-101559dafb09">

Reply:
I [left a comment in Figma](https://www.figma.com/design/qMpff4GuFUEcsMUkvacS3U?node-id=19910-110621&m=dev#1051618379), let's see what they say.